include/modules/hyprland/workspaces.hpp

```diff
@@ -70,14 +70,16 @@ class Workspace {
   std::string output() const { return m_output; };
   bool isActive() const { return m_isActive; };
   bool isSpecial() const { return m_isSpecial; };
-  bool isPersistent() const { return m_isPersistent; };
+  bool isPersistent() const { return m_isPersistentRule || m_isPersistentConfig; };
+  bool isPersistentConfig() const { return m_isPersistentConfig; };
```

Comment:
For consistency, maybe you could also add an `isPersistentRule()` function that returns the `m_isPersistentRule` boolean.

Reply:
Done. I didn't add it initially as it's not used and is a getter easy to add.